README.md

```diff
@@ -1,6 +1,64 @@
 # dhall-bhat
 Tasty meal of Dhall
 
+We currently require Dhall 1.18.0, and try to support newer releases as much as possible.
+
 ## Documentation
 
 Types as documentation: https://formationai.github.io/dhall-bhat/
+
+## Style
+
+Dhall is still a very young community, so the style is still evolving. Here’s how we’re currently doing things in dhall-bhat.
+
+### group expressions by type
```

Comment:
nit: I prefer these headings weren't lowercase

Reply:
I was thinking of these less as headings and more as `<dl>`, which Markdown doesn’t yet support.